src/main/java/org/testng/TestRunner.java

```diff
@@ -63,15 +63,14 @@
 public class TestRunner
     implements ITestContext, ITestResultNotifier, IThreadWorkerFactory<ITestNGMethod> {
 
-  public static final String DEFAULT_PROP_OUTPUT_DIR = "test-output";
+  private static final String DEFAULT_PROP_OUTPUT_DIR = "test-output";
```

Comment:
It is in the public api, so it could be used outside. Not a big break, but it may happen.